README.adoc

```diff
@@ -66,6 +66,24 @@ NOTE: The phone will need to be disconnected and reconnected for every build (`.
 Any other actions can be done with the `run` action.
 `./dockerize.sh run` and `npm run` are equivalent except that the former runs the environment in docker.
 
+==== Building a Signed APK
+
+. Ensure the image is initalized with `./dockerize.sh init` and dependencies installed with `./dockerize.sh install`.
+
+. Copy `gradle.properties.tmpl` in the root directory to `gradle.properties`.
+`gradle.properties` is the file that will actually be used.
+Fill in the `AGRITRADER_DEPLOY_UNSTABLE_STORE_PASSWORD` and `AGRITRADER_DEPLOY_UNSTABLE_KEY_PASSWORD` fields with the password.
+Ask someone for the password, it is not version controlled.
++
+IMPORTANT: Be very careful not to commit the passwords to version control.
```

Comment:
Thank you for putting this in, as it is probably something I would have inadvertently done.